### PR TITLE
Init meeting-form

### PR DESCRIPTION
**Pull request by MarioPierides**
In this pull request I have added a form to create a new meeting - by using React UseState
The form adds a new meeting to the meeting list.

Note: The form and the list are in the same page for now. I will render them in different pages in the future, I'm just not there yet. No styled-components included until now.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,6 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "test": "react-scripts test",
```
Comment (F-Kirchhoff) [inline package.json:10]: Why did you delete this line?

File: pages/index.js [hunk 2]
```diff
@@ -1,10 +1,71 @@
...
 
 export default function Home() {
-  return <Title>My page</Title>;
+  const [meetings, setMeetings] = useState([]);
+
+  function handleSubmit(event) {
+    console.log(event);
+    event.preventDefault();
+
+    const titel = event.target.elements.titel.value;
```
Comment (F-Kirchhoff) [inline pages/index.js]: Please stay english: 'title' ;)

File: pages/index.js [hunk 3]
```diff
@@ -1,10 +1,71 @@
...
+    const location = event.target.elements.location.value;
+    const activity = event.target.activities.value;
+
+    setMeetings(prevState => [...prevState, { titel, location, activity }]);
```
Comment (F-Kirchhoff) [inline pages/index.js]: Wow 😲 thats quite advanced!

File: pages/index.js [hunk 4]
```diff
@@ -1,10 +1,71 @@
...
+
+    console.log(titel);
+    console.log(location);
+    console.log(activity);
```
Comment (F-Kirchhoff) [inline pages/index.js]: Please remove console.logs before you start a pullrequest ;) 🕵️

File: pages/index.js [hunk 5]
```diff
@@ -1,10 +1,71 @@
...
+          </select>
+        </div>
+        <div>
+          <label>Aktivitäten</label>
+          <select name="activities">
+            <option value="">Bitte auswählen</option>
+            <option value="gassigehen">Gassi gehen</option>
+            <option value="sport">Sport</option>
+            <option value="otherthings">Sonstiges</option>
+          </select>
```
Comment (F-Kirchhoff) [inline pages/index.js:50]: Looking very good, I only would suggest to use the singular here, to stay consistent (you use location in the upper one and not locations)

File: pages/index.js [hunk 6]
```diff
@@ -1,10 +1,71 @@
...
 export default function Home() {
-  return <Title>My page</Title>;
+  const [meetings, setMeetings] = useState([]);
+
+  function handleSubmit(event) {
+    console.log(event);
+    event.preventDefault();
+
+    const titel = event.target.elements.titel.value;
+    const location = event.target.elements.location.value;
```
Comment (F-Kirchhoff) [inline pages/index.js]: In a Form component the inputs with a `name` attribute are accessible without the `.elements` object. so you can omit the `.elements` here as well.

File: pages/index.js [hunk 7]
```diff
@@ -1,10 +1,71 @@
...
   font-size: 50px;
   color: ${({ theme }) => theme.colors.primary};
 `;
 
 export default function Home() {
-  return <Title>My page</Title>;
+  const [meetings, setMeetings] = useState([]);
+
+  function handleSubmit(event) {
+    console.log(event);
```
Comment (F-Kirchhoff) [inline pages/index.js:14]: Please delete all console.logs 🕵️

File: pages/index.js [hunk 8]
```diff
@@ -1,10 +1,71 @@
...
+
+        <div>
+          <button type="submit">Erstellen</button>
+        </div>
+      </form>
+
+      <ul>
+        {meetings.map(meeting => {
+          return (
+            <li>
```
Comment (F-Kirchhoff) [inline pages/index.js]: Here is the problem with vercel: you need to give each list item a unique key prop. 
```suggestion
            <li key={someUniqueKey}>
```
You must not use the index of the meeting but should implement an id in your meeting objects:
```
{
  title,
  location,
  activity,
  id: "1234"
}
```
Hint: you can use `nanoid` for that. You can find it in google.

File: pages/index.js [hunk 9]
```diff
@@ -1,10 +1,71 @@
...
+              <div>{meeting.titel}</div>
+              <div>{meeting.location}</div>
+              <div>{meeting.activity}</div>
```
Comment (F-Kirchhoff) [inline pages/index.js]: Please use semantic html here, for example a p tag or a span.

File: pages/meetings.js [hunk 10]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export default function Meetings() {
+  return <Title>Meetings</Title>;
+}
```
Comment (F-Kirchhoff) [inline pages/meetings.js]: this page does look empty, please remove it if it does not serve a purpose yet. You can add it in later on.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Wow, I am amazed Mario very clean code all in all. I added some suggestions for semantic and clean code, and you need to add a key to each component in your list when you map your meetings. Oh and please delete your hooks folder with the empty files in it.
But besides that very good job for your first PR! 🚀

File: pages/index.js [hunk 11]
```diff
@@ -14,30 +14,26 @@ export default function Home() {
     console.log(event);
```
Comment (oy-gen) [inline pages/index.js:14]: console.log in line 14 is still there, please delete.

**Review by oy-gen — COMMENTED** [review]
Please delete console.log in line 14. Everything else LGTM.

**Pull request merged**